extensions/wikia/Recirculation/RecirculationApiController.class.php

```diff
@@ -57,11 +57,19 @@ public function getCakeRelatedContent() {
 
 		$limit = trim( $this->request->getVal( 'limit' ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "limit" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)